runtests.py

```diff
@@ -2,12 +2,28 @@
 
 # Django must be set up before we import our libraries and run our tests
 
+import os
 import sys
 
 import django
 from django.conf import settings
 
 
+if os.environ.get('TEST_WITH_REDIS'):
```

Comment:
Another alternative would be to have one of the `env` values above be `TEST_WITH_REDIS=` (empty string)